### PR TITLE
Added Safari Extension Build Script

### PR DESCRIPTION
**Pull request by Camji55**
Build script allows this extension to be built for Safari.

This extension cannot be distributed without having it Notarized by Apple, or distributed in the Mac App Store. Let me know if you need help with either. In the meantime, users who have Xcode installed on their macOS system can build and run this themselves.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
   "main": "src/gitpodify.js",
   "scripts": {
     "build": "yarn clean && npx tsc && yarn webpack",
+    "build:safari": " yarn install && yarn build && yarn package && xcrun safari-web-extension-converter . --app-name Gitpod --bundle-identifier io.gitpod.Gitpod",
```
Comment (spoenemann) [inline package.json:10]: I'd keep `yarn install` as a separate step and rather not include it in this build script.

**Review by spoenemann — COMMENTED** [review]
This requires Xcode 12 Beta, correct?

I get
```
xcrun: error: unable to find utility "safari-web-extension-converter", not a developer tool or in PATH
```
with Xcode 11.6.

File: package.json (hunk 1 above)
Comment (Camji55) [reply package.json:10]: Cool, I'll update it. And correct Xcode 12 is required for the `safari-web-extension-converter`. I'll update the readme to note this.

Comment (Camji55) [reply package.json]: Fixed 😄

**Comment by spoenemann** [comment]
Thanks for the contribution! Unfortunately I don't have Xcode 12 yet. I'll get back to this when it has been released, unless someone else with the beta version installed can verify this.

**Comment by Camji55** [comment]
@spoenemann Xcode 12 GM was released today https://developer.apple.com/services-account/download?path=/Developer_Tools/Xcode_12_GM_seed/Xcode_12_GM_seed.xip.

Full release is expected tomorrow.

**Comment by Camji55** [comment]
@spoenemann Xcode 12 has been released to the public https://apps.apple.com/us/app/xcode/id497799835?mt=12

**Comment by spoenemann** [comment]
Installing... 😴

**Comment by spoenemann** [comment]
I've built the extension and enabled "Allow Unsigned Extensions". I can see it in the Extensions tab in the Safari Preferences. But I don't see any Gitpod button on GitHub repositories. What else might have gone wrong?

**Comment by Camji55** [comment]
@spoenemann If you right click the Safari Toolbar and click "Customize Toolbar", does it appear in there?

**Comment by Camji55** [comment]
Also make sure you're using the latest version of Safari, Safari 14 which adds the support for the universal extensions.

File: README.md [hunk 2]
```diff
@@ -7,14 +7,39 @@ This is the browser extension for Gitpod, supporting Chrome ([Chrome Web Store](
...
+First, run the following to install dependencies and build the web extension:
+
 ```
 yarn install && yarn build && yarn package
 ```
 
+Then run the `build:safari` command to build the Safari extension around the web extension:
+
+```
+npm run build:safari
```
Comment (spoenemann) [inline README.md:29]:
```suggestion
yarn build:safari
```

**Review by spoenemann — COMMENTED** [review]
Ok, it works after adding the button to the toolbar and clicking it.

A minor comment below.

File: README.md (hunk 2 above)
Comment (Camji55) [reply README.md]: Done 😄

**Review by spoenemann — APPROVED** [review]
Thank you!

**Pull request merged**